02.create-and-explore/04.entities/06.text-entities/docs.md

```diff
@@ -66,10 +64,10 @@ To change the background and text colors, in the **Properties** tab:
 
 1. Scroll down to the **Text color** and **Background color** properties.
 2. Change the RGB values to the following:
-   - Text color- Black (Red: 0, Green:0, Blue: 0)
-   - Background color - Green (Red: 0, Green:255, Blue: 0).
+   - Text color- Black (Red: 255, Green:0, Blue: 0)
```

Comment:
In the picture, the pink text color has a Blue: 255 value as well.  Can you confirm?

Reply:
and it says text Color - Black